pkg/graph/schema.graphql

```diff
@@ -1457,6 +1462,8 @@ type Query {
   @hasAnyRole(roles: [MINT_USER, MINT_MAC])
   existingModelLink(id: UUID!): ExistingModelLink!
   @hasAnyRole(roles:[MINT_USER, MINT_MAC])
+  searchChanges(filters: [SearchFilter!], limit: Int!, offset: Int!): [ChangeTableRecord!]!

```

Comment:
Maybe this should be named something more expressive. Something like searchFilteredChanges? Thoughts or ideas?

Reply:
searchChanges works for me! I think the params make it fairly clear that you provide a filter, so I think that works.